src/lib/sidebar/ListMode.svelte

```diff
@@ -55,6 +105,15 @@
 <hr />
 
 {#each Object.keys($gjSchemeCollection.schemes) as scheme_reference}
-  <PerSchemeControls {scheme_reference} />
+  <PerSchemeControls {scheme_reference}>
+    {#if Object.keys($gjSchemeCollection.schemes).length > 1}
+      <SecondaryButton on:click={moveSchemeInList(scheme_reference, -1)}>
```

Comment:
```suggestion
      <SecondaryButton on:click={() => moveSchemeInList(scheme_reference, -1)}>
```
then you don't need to return a callback from the function

Reply:
I don't think this works?

Reply:
Or at least I don't understand it, so for now I'm gonna leave this change